codemetrics.el

```diff
@@ -255,10 +256,14 @@ details.  Optional argument DEPTH is used for recursive depth calculation."
   (codemetrics--ensure-ts
     (let* ((mode (or mode major-mode))
            (rules (codemetrics--rules mode))
-           (nested-level)  ; this is the "starting" nested level
+           ;; Collection of nesting levels
+           (nested-depths '())
```

Comment:
Just leave it as `(nested-depths)`, which is a new, preferable way to declare an empty list.

Reply:
Ah, did not know that 🙂 Fixed in latest commit.